man/oci-image-tool.1.md

```diff
@@ -0,0 +1,43 @@
+% OCI(1) OCI-IMAGE-TOOL User Manuals
+% OCI Community
+% JULY 2016
+# NAME
+oci-image-tool \- OCI (Open Container Initiative) image tools
+
+# SYNOPSIS
+**oci-image-tool** [OPTIONS] COMMAND [arg...]
+
+**oci-image-tool** [--help|-v|--version]
+
+# DESCRIPTION
+oci-image-tool is a collection of tools for working with the [OCI image specification](https://github.com/opencontainers/image-spec).
+
+
+# OPTIONS
+**--help**
+  Print usage statement.
+
+**--log-level**=LEVEL
+  Log level (panic, fatal, error, warn, info, or debug) (default: "error").
+
+**-v**, **--version**
+  Print version information.
+
+# COMMANDS
+**validate**
+  Validate one or more image files
+  See **oci-image-tool-validate**(1) for full documentation on the **validate** command.
+
+**unpack**
+  Unpack an image or image source layout
+  See **oci-image-tool-unpack**(1) for full documentation on the **unpack** command.
+
+**bundleCreate**
```

Comment:
Let's avoid camelCase in our command designs. This should employ `<object> <verb>` or use dashes `bundle-create`.